index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
+  </head>
+  <body>
+    <form action="contactForm.php" method="POST" class="contact-form">
+
+  <div class="cf-item">
+    <label for="fname">First Name</label>
+    <input class="cf-input" type="text" id="fname" name="firstname" placeholder="Your name..">
+  </div>
+
+  <div class="cf-item">
+    <label for="lname">Last Name</label>
+    <input class="cf-input" type="text" id="lname" name="lastname" placeholder="Your last name..">
+  </div>
+
+  <div class="cf-item">
+    <label for="email">Email</label>
+    <input class="cf-input" type="text" id="email" name="email" placeholder="Email@email.com">
+  </div>
+
+  <div class="cf-item">
+    <label for="phone">Phone</label>
+    <input class="cf-input" type="text" id="phone" name="phone" placeholder="+44 01522 808080">
+  </div>
+
+  <div class="cf-functionality-wrap">
+
+    <div class="cf-item">
+      <label for="functionality1">Functionality1</label>
+      <input class="cf-input" type="checkbox" id="functionality1" name="functionality1" value="function 1">
+    </div>
+
+    <div class="cf-item">
+      <label for="functionality2">Functionality2</label>
+      <input class="cf-input" type="checkbox" id="functionality2" name="functionality2" value="function 2">
+    </div>
+
+    <div class="cf-item">
+      <label for="functionality3">Functionality3</label>
+      <input class="cf-input" type="checkbox" id="functionality3" name="functionality3" value="function 3">
+    </div>
+
+    <div class="cf-item">
+      <label for="functionality4">Functionality4</label>
+      <input class="cf-input" type="checkbox" id="functionality4" name="functionality4" value="function 4">
+    </div>
+
+  </div>
+
+  <div class="cf-item">
+    <label for="implementationDate">Implementation Date</label>
+    <input class="cf-input" type="date" id="implementationDate" name="Implementation Date">
+  </div>
+
+  <div class="cf-item">
+    <label for="implementationDate">Project Details</label>
+    <textarea class="cf-input" id="subject" name="subject" placeholder="Project Details"></textarea>
+  </div>
+
+
+  <input class="cf-submit" type="submit" value="Submit">
```

Comment:
I think it's better semantics to use a `<button>` element instead of an `<input>` here.